src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -132,4 +134,16 @@ public void merge_twoRepoConfigs_success() throws ParseException, IOException {
                 actualConfigs.get(0).getAuthorAliasMap().hashCode());
         Assert.assertEquals(REPO_LEVEL_GLOB_LIST, actualConfigs.get(0).getIgnoreGlobList());
     }
+
+    @Test
+    public void test_defaultBranch() throws ParseException, IOException {
+        RepoConfiguration expectedConfig = new RepoConfiguration(TEST_REPO_BETA_LOCATION,
+                CsvParser.DEFAULT_BRANCH);
+        String input = String.format("-config %s", TEST_EMPTY_BRANCH_CONFIG_FOLDER);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+        List<RepoConfiguration> actualConfigs =
+                new RepoConfigCsvParser(((ConfigCliArguments) cliArguments).getRepoConfigFilePath()).parse();
+        Assert.assertEquals(1, actualConfigs.size());
```

Comment:
Put an empty line between the setup and assertion checks, so its easier for readers to know which is the setting up and which is the actual assertion cehcking.

Reply:
Empty line has been added